modulefiles/jet

```diff
@@ -22,7 +22,7 @@ module load hdf5/1.10.6
 module load netcdf/4.7.4
 
 module load bacio/2.4.1
-module load crtm/2.3.0
+module load crtm/2.4.0
```

Comment:
This will make GSI use the 2.4.0.
Did you test GSI with this crtm/2.4.0. library?

Reply:
No, I did not test GSI. Maybe it's OK to leave it as crtm/2.3.0 since UPP just uses the binary files, which are copied in from v2.4.0 by the updated scripts.